### DISCUSSION TITLE
How to update my certification when I use AZURE marketplace AMS with VM.

### DISCUSSION THREAD

**Discussion by williewhcheng** [ant-media/Ant-Media-Server#6418] [FAQs]
How to update my certification when I use AZURE marketplace AMS with VM.  

![image](https://github.com/ant-media/Ant-Media-Server/assets/160831376/d91b56ba-8031-46c5-ba6a-75112b12f832)

**Comment by muratugureminoglu** [comment]
Hi @williewhcheng 

Your certificate should have auto-renewed normally; the reason it didn't is usually because port 80 is closed. Can you check if this port is open. And then run `sudo /usr/local/antmedia/enable_ssl.sh -d your-domain.com` to trigger the certificate renewal process again. Please let me know if it does not work.
